integration/nwo/config.go

```diff
@@ -9,15 +9,16 @@ package nwo
 // Config holds the basic information needed to generate
 // fabric configuration files.
 type Config struct {
-	Organizations []*Organization `yaml:"organizations,omitempty"`
-	Consortiums   []*Consortium   `yaml:"consortiums,omitempty"`
-	SystemChannel *SystemChannel  `yaml:"system_channel,omitempty"`
-	Channels      []*Channel      `yaml:"channels,omitempty"`
-	Consensus     *Consensus      `yaml:"consensus,omitempty"`
-	Orderers      []*Orderer      `yaml:"orderers,omitempty"`
-	Peers         []*Peer         `yaml:"peers,omitempty"`
-	Profiles      []*Profile      `yaml:"profiles,omitempty"`
-	Templates     *Templates      `yaml:"templates,omitempty"`
+	Organizations    []*Organization `yaml:"organizations,omitempty"`
+	Consortiums      []*Consortium   `yaml:"consortiums,omitempty"`
+	SystemChannel    *SystemChannel  `yaml:"system_channel,omitempty"`
+	Channels         []*Channel      `yaml:"channels,omitempty"`
+	Consensus        *Consensus      `yaml:"consensus,omitempty"`
+	Orderers         []*Orderer      `yaml:"orderers,omitempty"`
+	Peers            []*Peer         `yaml:"peers,omitempty"`
+	Profiles         []*Profile      `yaml:"profiles,omitempty"`
+	Templates        *Templates      `yaml:"templates,omitempty"`
+	ChaincodeServers []*Chaincode    `yaml:"chaincode_servers,omitempty"`
```

Comment:
This is not part of the network configuration but specific to a small subset of tests. There's no need to have this here.